clerk/users.go

```diff
@@ -249,6 +249,27 @@ func (s *UsersService) DisableMFA(userID string) (*UserDisableMFAResponse, error
 	return &response, nil
 }
 
+type OAuthProviderToken struct {
+	Object         string      `json:"object"`
+	Provider       string      `json:"provider"`
+	Token          string      `json:"token"`
+	PublicMetadata interface{} `json:"public_metadata"`
+	Label          string      `json:"label"`
```

Comment:
The `label` attribute is optional, let's make it a pointer to a `string` instead.
```suggestion
	Label          *string      `json:"label"`
```